Sming/Wiring/WString.cpp

```diff
@@ -23,84 +23,86 @@
 /*  Constructors                             */
```

Comment:
I would prefer also the Wiring files to have their original coding style. But if it is too much of a hassle it is also ok for me to leave it like this.

Reply:
Have reverted style changes.